docs/src/main/sphinx/release/release-389.md

```diff
@@ -0,0 +1,46 @@
+# Release 389 (6 July 2022)
+
+## General
+
+* Improve performance of queries which contain a large number of aggregations
+  that use the `row` type for representing intermediate state (e.g. `SUM`). ({issue}`12762`)
+* Improve join performance when spill to disk is not used. The old behavior for
+  the spilling join operator can be restored by setting either the
+  `force_spilling_join` configuration property or the
+  `force-spilling-join-operator` session property to false. ({issue}`12618`)
+* Remove deprecated version of `ConnectorRecordSetProvider#getRecordSet`. ({issue}`13084`)
+* Fix query failure when no arguments are passed to a table function. ({issue}`12951`)
+* Fix error where a split keeps running after query failure due to an
+  uninterruptible JONI regular expression function. ({issue}`13064`)
+
+## Delta Lake connector
+
+* Improve native parquet writer performance for primitive types. ({issue}`13030`)
+* Fix error where the server does not start if the internal table snapshots
+  cache is disabled. ({issue}`13086`)
+
+## Elasticsearch connector
+
+* Add `raw_query` table function for full query pass-through to the connector. ({issue}`12324`)
```

Comment:
Is it really called raw_query ... as compared to query in the JDBC connectors .. if so .. why the inconsistency

Reply:
https://github.com/trinodb/trino/pull/12324#discussion_r908852229